### ISSUE TITLE
Why is the valid not loaded from the flow files

### ISSUE THREAD

**Issue by benghalensis** [eugenelyj/blinkvision_benchmark#1]
Please check out this line: [[Link](https://github.com/eugenelyj/blinkvision_benchmark/blame/b9cc745783e673027882876afb2fa52382298b66/flow_v1/pytorch_dataloader/dataset.py#L66)]

```
def load_flow(self, flowfile):
    flow_32bit = np.load(flowfile)
    # valid flag labels forward-backward checking
    flow, valid2D = flow_32bit[:,:,:2], flow_32bit[:,:,2]
    # we evaluate on all pixels
    valid2D = np.ones_like(valid2D)
    flow = np.transpose(flow, [2, 0, 1])
    return flow, valid2D
```

The valid2D is being overwritten in the dataset.

**Comment by eugenelyj** [comment]
@benghalensis Sorry for the late reply. If I remember correctly, the valid flag is assumed to always true because the data is synthetically rendered and It doesn't suffer the accuracy problem as the real data.

**Comment by eugenelyj** [comment]
The valid here means whether the pixel pass the forward-backward consistency checking, which is used to determine the occlusion. As a result, it is ignored because the dataloader here is for training.

**Comment by benghalensis** [comment]
Okay, thank you.
Closing this issue.
